data/ca/retired/Acquanetta-Warren-594ad199-1d69-4af4-9761-a5df048f8ada.yml

```diff
@@ -0,0 +1,13 @@
+id: ocd-person/594ad199-1d69-4af4-9761-a5df048f8ada
+name: Acquanetta Warren
+given_name: Acquanetta
+family_name: Warren
+email: awarren@fontana.org
+roles:
+- end_date: '2022-12-31'
```

Comment:
@bfossen-ce this is a duplicate from earlier in the pull. This person is also not retiring as they just won re-election